### PR TITLE
Fix Undefined variable/key warnings

### PR DESCRIPTION
**Pull request by murrant**
Just a random selection when I run discovery/polling on a handful of real devices.

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/discovery/cisco-vrf-lite.inc.php [hunk 1]
```diff
@@ -21,7 +21,7 @@
     $ids = [];
 
     // For the moment only will be cisco and the version 3
-    if ($device['os_group'] == 'cisco' && $device['snmpver'] == 'v3') {
+    if (isset($device['os_group']) && $device['os_group'] == 'cisco' && $device['snmpver'] == 'v3') {
         $mib = 'SNMP-COMMUNITY-MIB';
         $mib = 'CISCO-CONTEXT-MAPPING-MIB';
```
Comment (Jellyfrog) [inline includes/discovery/cisco-vrf-lite.inc.php:26]: These two lines looks like a bug :)

Comment (murrant) [reply includes/discovery/cisco-vrf-lite.inc.php:26]: haha, yeah, fix is to delete line 25, but oh well.

Comment (PipoCanaja) [reply includes/discovery/cisco-vrf-lite.inc.php:26]: cisco-vrf-lite looks like a bug :) I haven't seen any cisco device that runs this module and not standard VRF module so far ... even my oldies. I suppose this is mostly legacy code now.

**Review by PipoCanaja — APPROVED** [review]
LGTM

**Pull request merged**